pkg/operator/monitor/status.go

```diff
@@ -48,6 +49,13 @@ func (c *ClusterStatusChecker) CheckClusterStatus(namespacedName *types.Namespac
 
 		case <-time.After(c.interval):
 			c.checkStatus(namespacedName)
+			if err := EnableServiceMonitor(ref); err != nil {
+				logger.Errorf("monitor failed err %s", err.Error())
```

Comment:
@GowthamShanmugam Do we need to handle this error other than just logging it?

Reply:
ack, will fix this